From m.kozlowski@tuxland.pl  Wed Mar 11 14:28:28 2009
From: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Date: Tue,  3 Mar 2009 19:38:31 +0100
Subject: Staging: comedi: usbduxfast: remove .bss variable initialization
To: gregkh@suse.de
Cc: m.kozlowski@tuxland.pl
Message-ID: <1236105518-19565-2-git-send-email-m.kozlowski@tuxland.pl>


This patch removes explicit zeroing of usbduxfastsub
variable on init because it is in .bss section.

Signed-off-by: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Cc: Bernd Porr <BerndPorr@f2s.com>
Cc: Ian Abbott <abbotti@mev.co.uk>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: David Schleef <ds@schleef.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/usbduxfast.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- a/drivers/staging/comedi/drivers/usbduxfast.c
+++ b/drivers/staging/comedi/drivers/usbduxfast.c
@@ -1875,11 +1875,8 @@ static void __init init_usb_devices(void
 	 * they will become valid by the probe function
 	 * and then finally by the attach-function
 	 */
-	for (index = 0; index < NUMUSBDUXFAST; index++) {
-		memset(&(usbduxfastsub[index]), 0x00,
-			sizeof(usbduxfastsub[index]));
+	for (index = 0; index < NUMUSBDUXFAST; index++)
 		init_MUTEX(&(usbduxfastsub[index].sem));
-	}
 }
 
 /*
